docs/Operators.md

```diff
@@ -5146,6 +5151,40 @@ expect(node, inputs=[data, new_shape], outputs=[expanded],
 </details>
 
 
+<details>
+<summary>dim_negative</summary>
+
+```python
+node = onnx.helper.make_node(
+    'Expand',
+    inputs=['data', 'new_shape'],
+    outputs=['expanded'],
+)
+shape = [3, 3, 1]
+new_shape = [-1, 3, 6]
```

Comment:
Why can't we use new_shape = [1, 3, 6] instead?

Reply:
This test is for the case when we have -1 in expand shape input.

Reply:
I was using this to illustrate my previous question: why do we need this extension of the op, if we can use "1" instead? Am I misunderstanding something?

Reply:
We do not always know the dim value we want to expand to.
Let's say you have a tensor of size (?, ?, 1) and you wan to expand to (?, ?, 5) where ? could be unknown.

Reply:
Then, you just specify the new-shape to be (1, 1, 5) ... if you apply this to a tensor of size (M,N,1), won't you end up with the shape (M, N, 5)?

Reply:
No,  you cannot expand (M, N, 1) tensor to (1, 1, K)

Reply:
Bidirectional broadcast between the shapes (1, 1, K) and (M, N, 1) will produce the shape (M, N, K). And I believe Expand (the existing version) does bidirectional-broadcast, as far as I can see. So, what am I missing?

Reply:
My understanding from the existing documentation is that ONNX Expand differs from numpy's broadcast_to in that ONNX uses bidirectional broadcast, unlike numpy.

Reply:
This is more of an export issue, since for exporting PT or TF models that allow -1, we need to change the shape input tensor.

Reply:
I see. So, the semantics of "-1" will be the same as that of "1" (in ONNX)? (Not sure if PT or TF have the exact same semantics for 1 ... it does look odd to have two ways of saying the same thing.)